soccer/gameplay/evaluation/opponent.py

```diff
@@ -65,3 +66,137 @@ def get_closest_opponent(pos, direction_weight=0, excluded_robots=[]):
                 closest_dist = dist
 
     return closest_bot
+
```

Comment:
Moved this code from defense.py

Reply:
Do we think this is the long term best place for these sorts of things? I just don't want this file to become huge

Reply:
It is better than it being in defense.py. this is a big file though. I'll think about splitting up the file a bit

Reply:
I think having multiple files in this folder will probably be the best plan going forward. Still, don't know if we want an even better structure than that.